pkg/server/handler/v1alpha1/tenant.go

```diff
@@ -3,26 +3,26 @@ package v1alpha1
 import (
 	"context"
 	"encoding/json"
+	"reflect"
 
 	"github.com/caicloud/nirvana/log"
 	"k8s.io/api/core/v1"
 	core_v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/resource"
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/util/retry"
 
-	apiv1alpha1 "github.com/caicloud/cyclone/pkg/server/apis/v1alpha1"
+	api "github.com/caicloud/cyclone/pkg/server/apis/v1alpha1"
```

Comment:
Do not recommend this change, if there is another version of API, we still need to specify the version.

Reply:
`tenant.go` is under the `v1alpha1` folder, so it is less likely it needs another version of API.